frontend/public/kubevirt/components/cluster/cluster-overview.jsx

```diff
@@ -100,11 +108,37 @@ export class ClusterOverview extends React.Component {
         },
         loaded: false,
       },
+      capacityStats: {
+        stats: {
+          cpu: {
+            title: 'CPU',
```

Comment:
It would be nice if every stat would hold an information if it is loaded or not and Cluster Capacity would be then able to show loading only for not-loaded charts. Same as done in Details https://github.com/kubevirt/web-ui-components/blob/master/src/components/ClusterOverview/Details/ClusterDetails.js#L14

Reply:
Actually it is already there, I am just not reserving the space by a loader component nut filtering the stat out. Right, this can be changed.
https://github.com/kubevirt/web-ui-components/pull/281/files#diff-ba92f97bf5d88a5c4e65a1b9732e9b44R63

Reply:
ah, didnt notice that you filter not loaded stats. Good!